.github/workflows/publish_frontend.yml

```diff
@@ -8,7 +8,7 @@ env:
 
 jobs:
   build-and-push-frontend-image:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, linux]
```

Comment:
Same here

Reply:
Proxying no longer needed to access ghcr.io, but docker.io rate limits could still be hit --> mirror configuration added.